### PR TITLE
Review of Exercise 0, feel free to disregard if you don't agree

### PR DESCRIPTION
**Pull request by katie-day**

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -20,3 +19,5 @@ export const exerciseZeroFormSubmit = (submit) => {
         },
     };
 };
+
+// Meet me in ../reducers/exercise0.js once you've understood this file
```
Comment (katie-day) [inline src/actions/index.js:23]: Moved the "meet me in" comments to the bottom of the file, that way people have to read through the whole file before they get to it

File: src/components/ExerciseZeroForm.js [hunk 2]
```diff
@@ -39,19 +36,18 @@ const options = [
 ];
 
 // Here comes the JSX part
-// The form is a stateless component in which we just plug the onSubmit method
+// The form is a stateless component in which we just plug the onSubmit (handleSubmit) method
```
Comment (katie-day) [inline src/components/ExerciseZeroForm.js:39]: Just added the extra reference here about `onSubmit` but called `handleSubmit` in the component.

File: src/components/ExerciseZeroForm.js [hunk 3]
```diff
@@ -39,19 +36,18 @@ const options = [
...
-// That's what we are up to now. Meet me in /component/Form/InputField.js
+// It basically requires at least a name and a rendering component eg. InputField or SelectField.
 const ExerciseZeroForm = ({ handleSubmit, submitSucceeded, weather, mood }) => (
     <div>
         <div style={{ marginBottom: '10px' }}>
             <i>
                 Use this exercise as a reference for a working example.
                 Follow me through the file comments.<br />
-                Meet me in /index.js
+                To start meet me in /src/index.js
```
Comment (katie-day) [inline src/components/ExerciseZeroForm.js:50]: added `/src/` because this is now not in the root folder

**Comment by katie-day** [comment]
@vincentaudebert Will the exercises eventually all be merged into master? If not, there needs to be some documentation in the Master / Readme to say to switch to branch exercise-0 to begin :)

**Comment by vincentaudebert** [comment]
I can confirm that exercise-0 will be merge to master right now :)

**Pull request merged**